cluster_test.go

```diff
@@ -1454,47 +1451,36 @@ func TestParseClusterURL(t *testing.T) {
 		},
 	}
 
-	for i := range cases {
-		tc := cases[i]
-		t.Run(tc.test, func(t *testing.T) {
-			t.Parallel()
-
+	It("match ParseClusterURL", func() {
+		for i := range cases {
+			tc := cases[i]
 			actual, err := redis.ParseClusterURL(tc.url)
-			if tc.err == nil && err != nil {
-				t.Fatalf("unexpected error: %q", err)
-				return
-			}
-			if tc.err != nil && err == nil {
-				t.Fatalf("expected error: got %+v", actual)
-				return
-			}
-			if tc.err != nil && err != nil {
-				if tc.err.Error() != err.Error() {
-					t.Fatalf("got %q, expected %q", err, tc.err)
-				}
-				return
+			if tc.err != nil {
+				Expect(err).To(Equal(err))
```

Comment:
Minor suggestion, I usually use `MatchError` when matching errors, since it handles error unwrapping.

```suggestion
				Expect(err).To(MatchError(err))
```

Reply:
ok thanks for your suggestion.